api/src/main/java/org/pmiops/workbench/db/model/DbWorkspace.java

```diff
@@ -22,13 +26,17 @@
 import java.util.stream.Collectors;
 import org.apache.commons.lang3.builder.EqualsBuilder;
 import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.pmiops.workbench.db.model.DbFeaturedWorkspace.DbFeaturedCategory;
 import org.pmiops.workbench.model.BillingStatus;
 import org.pmiops.workbench.model.DisseminateResearchEnum;
 import org.pmiops.workbench.model.ResearchOutcomeEnum;
 import org.pmiops.workbench.model.SpecificPopulationEnum;
 import org.pmiops.workbench.model.WorkspaceActiveStatus;
 
 @Entity
+@SecondaryTable(
```

Comment:
Whoa! I don't think I've seen this before. This is great.

Reply:
Very cool!  I'm tryign another conversion: https://github.com/all-of-us/workbench/pull/8687